cmd/promtool/main.go

```diff
@@ -510,18 +510,22 @@ func parseTime(s string) (time.Time, error) {
 	return time.Time{}, fmt.Errorf("cannot parse %q to a valid timestamp", s)
 }
 
+var profilesList = map[string]string{
+	"/debug/pprof/profile":      "cpu.pb",
+	"/debug/pprof/trace":        "trace.pb",
```

Comment:
IIUC this will only capture 1 second of trace data. Do we want a higher default value and some option to tweak it?

Reply:
Yeah hard coding doesn't bother me.